### ISSUE TITLE
Conflicts with keyboard bankpin

### ISSUE THREAD

**Issue by blankaex** [SirGirion/GlobalFKeys#6]
Using this plugin to set keybinds on number keys (0~9) interferes with the "keyboard bankpin" option in the built-in "bank" plugin that allows you to enter your bank pin with your keyboard. As you can't switch interfaces on the bank pin menu, a quick fix would be to disable this plugin's functionality when the bank pin interface is open.

**Comment by rvbzz** [comment]
Agree. Disable the plugins functionality so i don't have to use my damn numpad to enter my bankpin please.
